features/environment_secret.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: UTF-8 -*-
 import os
 
-HIPCHAT_LOGIN = os.environ.get("HCLOGIN", "tim+test@bortnik.org")
-HIPCHAT_PASS = os.environ.get("HCPASS", "ntcn987654321")
+HIPCHAT_LOGIN = os.environ.get("HCLOGIN", "danilyuk@bigmir.net")
+HIPCHAT_PASS = os.environ.get("HCPASS", "vjhzxjrntcn")
```

Comment:
Any password exposed to open repository should be changed ASAP

Reply:
remove password